### PR TITLE
Enhance UI"Contacts backup" 

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Ref #880 

remaining open issue:

- [x] support multiple accounts

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
👍 code looks good to me

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky 
```
found errors: 75
found warnings: 853
previous errors: 75
previous warnings: 852
```

**Comment by mario** [comment]
Let me know when this is ready...

**Comment by AndyScherzinger** [comment]
rebase but untested + remaining open issues according to #880

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky can you check codacy please, for some reason something changed and whenever codacy reruns the code check the old comment get removed but now this is done under your user!

>  tobiasKaminsky deleted a comment from codacy-bot

**Comment by tobiasKaminsky** [comment]
This is now done.

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky warnings increased:
```
found errors: 73
found warnings: 855
previous errors: 73
previous warnings: 853
```

**Comment by tobiasKaminsky** [comment]
This is now because I deleted some obsolete strings and now lint is complaining that they still are in transifex.
We will have to "just" merge this and then we can delete those not needed translations in transifex (or is it doing this automatically?)

tl;dr: please review and ignore lint

**Comment by AndyScherzinger** [comment]
> or is it doing this automatically?

removed translation strings are automatically removed from all other language files :)

**Comment by AndyScherzinger** [comment]
👍 for merge
